assets/quick-order-list.js

```diff
@@ -262,60 +275,65 @@ if (!customElements.get('quick-order-list')) {
         this.querySelectorAll('quantity-input').forEach((qty) => {
           const debouncedOnChange = debounce((event) => {
             this.onChange(event);
-          }, ON_CHANGE_DEBOUNCE_TIMER);
+          }, 100);
```

Comment:
Because we are assuming the buyer will now click very quickly (or could), it makes sense for us to have this be a shorter amount. I spoke to Tyler before too and the previous amount was arbitrary

Reply:
Should this match the timeout for the queue in the bulk-add component?

https://github.com/Shopify/dawn/pull/3395/files#diff-7b699c3e0ddf5492f7762bfb759ca470eba4f3e8ff2696351106ef84404fc5f6R18

Reply:
I havent gotten to doing the quick add bulk work yet, but that will be in a follow up here https://github.com/orgs/Shopify/projects/6397/views/28?pane=issue&itemId=59790024 :)